src/locutus/api/terminology.py

```diff
@@ -214,4 +214,29 @@ def delete(self, id, code=None):
             "terminology": {"Reference": f"Terminology/{t.id}"}
         }
 
+        return (response, 200, default_headers)
+    
+class PreferredTerminology(Resource):
+    def get(self, id=None):
+        t = Term.get(id)
+
+        pref = t.get_preferred_terminology()
+
+        return (pref, 200, default_headers)
+        
+    def post(self, id):
+        """Add a `preferred_terminology` to a specific Terminology"""
+        body = request.get_json()
+        t = Term.get(id)
+        if "preferred_terminology" not in body:
+            return {"message": "preferred_terminology is required"}, 400
+
+        preferred_terminology = body["preferred_terminology"]
+
+        t.add_preferred_terminology(preferred_terminology=preferred_terminology)
+        response = {
+            "terminology": {"Reference being mapped": f"Terminology/{t.id}"},
```

Comment:
I'm inclined to think this should be:
response = {
  "id": t.id,
  "preferred_terminology": preferred_terminology,
}
If you want to return the reference details for the terminology you are changing, you should make it a formal reference (so "Reference being mapped" would be "reference"). But, I think id is fine.